### PR TITLE
Add a views for each posted color and status

### PR DESCRIPTION
**Pull request by henryrizzi**
Add in views for each individual status and color at `/view/c/(status_color)` and `/view/n/(status_name)`. This is at least in part to accommodate a greater amount of information being posted to the site.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ Algoth is simple service to send and retrieve "current status" messages. Keeps t
 ### URL
 
 `/` or `/<nickname>`
- 
+
 HTML of all current statuses at `/view`.
```
Comment (tym-xqo) [inline README.md:9]: Add nickname and color URL option note here.

**Review by tym-xqo — APPROVED** [review]
This is such a great improvement! 😃

File: README.md (hunk 1 above)
Comment (henryrizzi) [reply README.md:9]: Just added that into the readme. :)

Comment (henryrizzi) [reply README.md:9]: 🚀

**Pull request merged**